geomesa-blobstore/geomesa-blobstore-core/src/main/java/org/locationtech/geomesa/blob/core/impl/AccumuloGeoMesaBlobStore.java

```diff
@@ -0,0 +1,140 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+
+package org.locationtech.geomesa.blob.core.impl;
+
+import org.geotools.data.Query;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreParams;
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore;
+import org.locationtech.geomesa.blob.core.interop.GeoMesaBlobStore;
+import org.opengis.filter.Filter;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class AccumuloGeoMesaBlobStore implements GeoMesaBlobStore {
+
+    protected final AccumuloBlobStore accumuloBlobStore;
+
+    public AccumuloGeoMesaBlobStore(Map<String, Serializable> dataStoreParams) throws IOException {
+        AccumuloDataStore ds = genNewADS(dataStoreParams);
+        if (ds == null) {
+            throw new IOException("Error initializing AccumuloGeoMesaBlobStore");
+        } else {
+            accumuloBlobStore = new AccumuloBlobStore(ds);
+        }
+    }
+
+    public AccumuloGeoMesaBlobStore(String instanceId,
+                                    String tableName,
+                                    String zookeepers,
+                                    String user,
+                                    String password,
+                                    String auths) throws IOException {
+        Map<String, Serializable> dataStoreParams = new HashMap<>();
+        dataStoreParams.put(AccumuloDataStoreParams.instanceIdParam().key, instanceId);
+        dataStoreParams.put(AccumuloDataStoreParams.tableNameParam().key, tableName);
+        dataStoreParams.put(AccumuloDataStoreParams.zookeepersParam().key, zookeepers);
+        dataStoreParams.put(AccumuloDataStoreParams.userParam().key, user);
+        dataStoreParams.put(AccumuloDataStoreParams.passwordParam().key, password);
+        dataStoreParams.put(AccumuloDataStoreParams.authsParam().key, auths);
+        AccumuloDataStore ds = genNewADS(dataStoreParams);
```

Comment:
I tried that, but intellij was not happy with that unless I removed the `final` from the accumuloblobstore further up the file.